File: 정수빈/Programmers/Level2/점프와순간이동.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12980
# 설명: 문제는 복잡하지만 답은 간단하다. 숫자를 2진수로 변환한 후 1의 갯수를 세면 된다.

def solution(n):
    return str(format(n, "b")).count("1")

File: 장세훈/Programmers/Level2/점프와순간이동.java
    class 점프와순간이동 {
    /*
        https://school.programmers.co.kr/learn/courses/30/lessons/12980
     */

        public int solution(int n) {
            int answer = 0;

            while (n != 0) {
                if (n % 2 != 0)
                    answer++;
                n /= 2;
            }
            return answer;
        }
    }